src/tools/tidy/src/features.rs

```diff
@@ -35,9 +44,10 @@ impl fmt::Display for Status {
 #[derive(Debug, Clone)]
 pub struct Feature {
     pub level: Status,
-    pub since: String,
+    pub since: Option<Version>,
     pub has_gate_test: bool,
     pub tracking_issue: Option<u32>,
+    pub group: Option<String>,
```

Comment:
Hmm; you don't actually use `.group` anywhere so it seems to me that unnecessary cloning is happening as a result. Do you want to use this for something later on? Otherwise I think it's best to drop this and introduce it when it is needed.